mods/ra/maps/exodus/rules.yaml

```diff
@@ -0,0 +1,185 @@
+Player:
+	MissionObjectives:
+	PlayerResources:
+		DefaultCash: 5000
```

Comment:
The whole `Player` block is redundant, it already has `MissionObjectives` and 5000 already is the `DefaultCash` default value.